### PR TITLE
Remove channel containers from DOM after quitting network

### LINKED ISSUE #1533: Qutting network does not remove channel containers from DOM
It only removes it from the sidebar here: https://github.com/thelounge/lounge/blob/59a5f6237cb3bc84637ca6c62e299d3179bbad2f/client/js/socket-events/quit.js#L9

### PR DESCRIPTION
**Pull request by PolarizedIons**
Closes #1533

### CODE REVIEW

File: client/js/socket-events/quit.js [hunk 1]
```diff
@@ -1,12 +1,20 @@
...
 const sidebar = $("#sidebar");
 
 socket.on("quit", function(data) {
 	const id = data.network;
-	sidebar.find("#network-" + id).remove();
+	const network = sidebar.find("#network-" + id);
+
+	network.children().each(function() {
+		// this = child
+		chat.find(this.getAttribute("data-target")).remove();
```
Comment (xPaw) [inline client/js/socket-events/quit.js]: Use `$(this).data("target")`

File: client/js/socket-events/quit.js [hunk 2]
```diff
@@ -1,12 +1,20 @@
...
+const chat = $("#chat");
 const socket = require("../socket");
 const sidebar = $("#sidebar");
 
 socket.on("quit", function(data) {
 	const id = data.network;
-	sidebar.find("#network-" + id).remove();
+	const network = sidebar.find("#network-" + id);
+
+	network.children().each(function() {
```
Comment (xPaw) [inline client/js/socket-events/quit.js]: I'd add `.chan` selector in `children`.

File: client/js/socket-events/quit.js (hunk 1 above)
Comment (PolarizedIons) [reply client/js/socket-events/quit.js]: Was thinking it was unnecessary to use jQuery for this, but sure :+1:

File: client/js/socket-events/quit.js (hunk 2 above)
Comment (PolarizedIons) [reply client/js/socket-events/quit.js]: :+1:

**Pull request merged**